<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdBlockDialog</class>
 <widget class="QWidget" name="AdBlockDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>424</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="m_cbEnable">
     <property name="text">
      <string>Enable AdBlock</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="m_tcSubscriptions">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="m_tabPredefinedLists">
      <attribute name="title">
       <string>Filter lists (list per line)</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPlainTextEdit" name="m_txtPredefined"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabCustomFilters">
      <attribute name="title">
       <string>Custom filters</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="m_txtCustom"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
